<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="1" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-12-28T13:36:15 IST" name="Suite" finished-at="2022-12-28T13:36:33 IST" duration-ms="18276">
    <groups>
    </groups>
    <test started-at="2022-12-28T13:36:15 IST" name="Test" finished-at="2022-12-28T13:36:33 IST" duration-ms="18276">
      <class name="scripts.Testcase2">
        <test-method is-config="true" signature="openApp()[pri:0, instance:scripts.Testcase2@636be97c]" started-at="2022-12-28T13:36:15 IST" name="openApp" finished-at="2022-12-28T13:36:19 IST" duration-ms="4009" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApp -->
        <test-method signature="tc2()[pri:0, instance:scripts.Testcase2@636be97c]" started-at="2022-12-28T13:36:19 IST" name="tc2" finished-at="2022-12-28T13:36:19 IST" duration-ms="331" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "Object.hashCode()" because "key" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "Object.hashCode()" because "key" is null
at java.base/java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:936)
at java.base/java.util.Properties.getProperty(Properties.java:1141)
at genaricLib.PropertyFile.getPropertydata(PropertyFile.java:15)
at scripts.Testcase2.tc2(Testcase2.java:25)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc2 -->
        <test-method is-config="true" signature="closeApp(org.testng.ITestResult)[pri:0, instance:scripts.Testcase2@636be97c]" started-at="2022-12-28T13:36:19 IST" name="closeApp" finished-at="2022-12-28T13:36:23 IST" duration-ms="3995" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc2 status=FAILURE method=Testcase2.tc2()[pri:0, instance:scripts.Testcase2@636be97c] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- scripts.Testcase2 -->
      <class name="scripts.Testcase1">
        <test-method is-config="true" signature="openApp()[pri:0, instance:scripts.Testcase1@1dde4cb2]" started-at="2022-12-28T13:36:23 IST" name="openApp" finished-at="2022-12-28T13:36:27 IST" duration-ms="3443" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openApp -->
        <test-method signature="tc1()[pri:0, instance:scripts.Testcase1@1dde4cb2]" started-at="2022-12-28T13:36:27 IST" name="tc1" finished-at="2022-12-28T13:36:33 IST" duration-ms="5954" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tc1 -->
        <test-method is-config="true" signature="closeApp(org.testng.ITestResult)[pri:0, instance:scripts.Testcase1@1dde4cb2]" started-at="2022-12-28T13:36:33 IST" name="closeApp" finished-at="2022-12-28T13:36:33 IST" duration-ms="499" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=tc1 status=SUCCESS method=Testcase1.tc1()[pri:0, instance:scripts.Testcase1@1dde4cb2] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeApp -->
      </class> <!-- scripts.Testcase1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
